src/frontend/src/optimizer/plan_node/logical_topn.rs

```diff
@@ -116,12 +115,6 @@ impl LogicalTopN {
         &self.core.group_key
     }
 
-    /// Infers the state table catalog for [`StreamTopN`] and [`StreamGroupTopN`].
-    pub fn infer_internal_table_catalog(&self, vnode_col_idx: Option<usize>) -> TableCatalog {
-        self.core
-            .infer_internal_table_catalog(&self.base, vnode_col_idx)
-    }
-
```

Comment:
Removed because `LogicalTopN` shouldn't infer table catalog for stream, and this method is not used anywhere.